Corresponding version in R of the sirsi_basic_model. Issue #18

fellipedoprado opened the pull request:
Contains:
1 - a XML file
2 - an input file in R
3 - the model file.
Fixes #18

fellipedoprado committed:
Corresponding version in R of the sirsi_basic_model Issue #18

Contains:
1 - a XML file
2 - an input file in R
3 - the model file.
Fixes #18

tiagofml reviewed (commented):
I will make the required changes.

tiagofml reviewed (changes requested):
In a first look, there are some required changes:
- the dependency of deSolve must be declared in the XML (see r_sir_basic_model.xml)
- use lower case for "true" in XML but upper case in the R input file (later we will discuss about how to deal with this)
- (please check) the parameters, their values and corresponding comments must be strictly equal in the XML and the r input. The input will be generated by the XML, thus, the values of parameters and other attributes in XML will be used to generate the input source code, and must be strictly equal.
- remove the R history and other unnecessary files (e.g. generated results such as .csv files)
- use lower case in the extension and file names

tiagofml committed:
Develop a corresponding version in R of the SIRSI_basic_model. Issue #18

Develop a corresponding version in R of the SIRSI_basic_model.
An error with dependencies was corrected in XML and input files. And
small changes in XML, input and model files were made related to
upper/lower case and source-coding format (e.g. blank spaces).
Fixes #18
